src/main/js/controls/Gantt/helpers/creationHelpers.js

```diff
@@ -310,18 +311,36 @@ const scaleGraph = (scale, config) => {
  * @private
  * @param {object} config - config object derived from input JSON
  * @param {d3.selection} canvasSVG - d3 selection node of canvas svg
- * @returns {object} d3 svg path
+ * @returns {undefined} return nothing
  */
-const createDefs = (config, canvasSVG) =>
-    canvasSVG
-        .append("defs")
+const createDefs = (config, canvasSVG) => {
+    const defsElement = canvasSVG.append("defs");
+
+    defsElement
         .append("clipPath")
         .attr("id", config.clipPathId)
         .append("rect")
         .attr(constants.X_AXIS, getXAxisXPosition(config))
         .attr(constants.Y_AXIS, getYAxisYPosition(config))
         .attr("width", getXAxisWidth(config))
         .attr("height", getYAxisHeight(config));
+
+    if (
+        config.pan !== undefined &&
```

Comment:
Ah you are doing it. 💯 
This is what I meant, since we already have the `settingsDictionary` within config, we could just listen to what that new property would say instead of rechecking if the panning is enabled.

Reply:
I will make the change